data/net/minecraft/world/level/SignalGetter.mapping

```diff
@@ -1,19 +1,50 @@
 CLASS net/minecraft/world/level/SignalGetter
 	METHOD getBestNeighborSignal (Lnet/minecraft/core/BlockPos;)I
+		COMMENT Returns the highest redstone signal the given position receives from neighboring blocks.
 		ARG 1 pos
 	METHOD getControlInputSignal (Lnet/minecraft/core/BlockPos;Lnet/minecraft/core/Direction;Z)I
+		COMMENT Returns the control signal emitted from the given position in the given direction.
+		COMMENT If {@code diodesOnly} is {@code true}, this method returns the direct signal emitted if
+		COMMENT and only if this position is occupied by a diode (i.e. a repeater or comparator).
+		COMMENT Otherwise, if this position is occupied by a redstone block, this method will return the
```

Comment:
Just to clarify, when you say "redstone block", you mean the block made from 9 redstone dust which emits a signal strength of 15? Might be good to `{@linkplain}` to `Blocks#REDSTONE_BLOCK` to clarify that in the comment.

Reply:
That's correct. Yeah sounds good, I'll go add a link to that comment.